.code-samples.meilisearch.yaml

```diff
@@ -548,3 +548,4 @@ post_dump_1: |-
   client.create_dump().await.unwrap();
 get_dump_status_1: |-
   client.get_dump_status("20201101-110357260").await.unwrap();
+  
```

Comment:
Can you remove the whitespace and add a new line at the end of the file?

To avoid this kind of error, you can download the editorconfig plugin, depending on your text editor. After downloading it, every time you will save your file, it will automatically apply the basic style rules to your file (removing trailing whitespaces, adding a new line at the end of file...).
See more about editorconfig: https://editorconfig.org/#overview
And the editorconfig file of this repo: https://github.com/meilisearch/meilisearch-rust/blob/master/.editorconfig

Reply:
I though it was standard to set a empty line at the end of every file. Removing...